tests/encode-utf8.html

```diff
@@ -10,13 +10,95 @@
 const inputString = 'I \u{1F499} streams';
 const expectedOutputBytes = [73, 32, 240, 159, 146, 153, 32, 115, 116, 114, 101,
                              97, 109, 115];
+const astralCharacter = '\u{1F499}';
```

Comment:
looks it's a blue heart. what does astral mean here?

Reply:
https://www.quora.com/Why-are-unicode-characters-outside-the-BMP-called-astral

I actually originally called the constant "plain1Character" but it was cumbersome and hard to read, so I looked for a better name.

I will add a clarifying comment.

Reply:
aha, thanks for the explanation. I didn't know that.

Reply:
how about supplementary?